menu/navigation.json

```
@@ -3886,4 +3890,4 @@
     ],
     "label": "Product Categories"
   }
-]
+]
```

add line break at the end otherwise the build fails
```suggestion
]

```